### ISSUE TITLE
PostgreSQL: Metrics Database Overview dashboard shows no data

### ISSUE THREAD

**Issue by ycombinator** [elastic/integrations#982]
As part of testing the PostgreSQL package with the 7.13.0 release, I am finding that the "[Metrics PostgreSQL] Database Overview" dashboard is not showing any data: 

<img width="1721" alt="Screen Shot 2021-05-10 at 3 30 42 PM" src="https://user-images.githubusercontent.com/51061/117732664-2269a080-b1a5-11eb-988a-8df4f41c953b.png">

<img width="1721" alt="Screen Shot 2021-05-10 at 3 30 45 PM" src="https://user-images.githubusercontent.com/51061/117732658-20074680-b1a5-11eb-8b88-5de87b814122.png">

After I setup Elastic Agent to collect metrics (and logs) from PostgreSQL, I created a database `foo` and a table `bar` in it.

```
postgres=# CREATE DATABASE foo;
CREATE DATABASE

postgres=# \c foo
You are now connected to database "foo" as user "postgres".

foo=# CREATE TABLE bar (); 
CREATE TABLE

foo=# \dt
        List of relations
 Schema | Name | Type  |  Owner
--------+------+-------+----------
 public | bar  | table | postgres
(1 row)

foo=# select * from bar;
--
(0 rows)

foo=# ^D\q
```

 I refreshed the dashboard a couple of minutes later but still don't see any data.

I don't know enough about PostgreSQL to know if I should be doing something more to see data on this dashboard.

**Comment by ycombinator** [comment]
cc: @sayden @masci @mtojek

**Comment by ycombinator** [comment]
Various data streams for the `postgresql` integration are present.

<img width="1721" alt="Screen Shot 2021-05-10 at 3 38 35 PM" src="https://user-images.githubusercontent.com/51061/117733092-d66b2b80-b1a5-11eb-85da-461617fc3412.png">

**Comment by ChrsMark** [comment]
@ycombinator did you check if metrics used in visualisations are being populated? It happened to me with another integration that some visualisations had wrong datastream filters and hence even if the metrics were being populated the visualisations were empty.

**Comment by mtojek** [comment]
I quickly booted up the Docker image with PostgreSQL and found the problem around SSL:

<img width="1977" alt="Zrzut ekranu 2021-05-11 o 15 12 50" src="https://user-images.githubusercontent.com/14044910/117821004-6aeb9180-b26b-11eb-81cb-5c7b792d2183.png">

**Comment by ycombinator** [comment]
Yep, this was user error. I needed to set the `username` and `password` advanced options + add `?sslmode=disable` to the `host` setting in the Integration configuration UI.
